vagrant/UPGRADING.md

```diff
@@ -0,0 +1,49 @@
+UPGRADING
```

Comment:
do we need all these files? if this is an external dependency, could we provide a simple shell script to fetch dependencies instead?